Gemfile

```diff
@@ -56,6 +57,7 @@ gem 'oauth2', '~> 1.4.7' # Used for Stripe Connect
 
 gem 'pagy', '~> 4.8'
 
+gem 'gitlab-omniauth-openid-connect', gitlab: 'gitlab-org/gitlab-omniauth-openid-connect', require: 'omniauth_openid_connect'
```

Comment:
There's a published gem for this repo, we could drop the custom gitlab source and just use:
```rb
gem 'gitlab-omniauth-openid-connect', require: 'omniauth_openid_connect'
```